Extensions/Lighting/JsExtension.js

```diff
@@ -380,37 +337,22 @@ module.exports = {
      * This is called to update the PIXI object on the scene editor
      */
     RenderedLightObjectInstance.prototype.update = function () {
-      this._pixiObject.shader.uniforms.center = new Float32Array([
-        this._instance.getX(),
-        this._instance.getY(),
-      ]);
-
-      this._vertexBuffer[0] = this._instance.getX() - this._radius;
-      this._vertexBuffer[1] = this._instance.getY() + this._radius;
-      this._vertexBuffer[2] = this._instance.getX() + this._radius;
-      this._vertexBuffer[3] = this._instance.getY() + this._radius;
-      this._vertexBuffer[4] = this._instance.getX() + this._radius;
-      this._vertexBuffer[5] = this._instance.getY() - this._radius;
-      this._vertexBuffer[6] = this._instance.getX() - this._radius;
-      this._vertexBuffer[7] = this._instance.getY() - this._radius;
-
-      this._pixiObject.geometry
-        .getBuffer('aVertexPosition')
-        .update(this._vertexBuffer);
+      this._pixiObject.position.x = this._instance.getX();
+      this._pixiObject.position.y = this._instance.getY();
     };
 
     /**
      * Return the width of the instance, when it's not resized.
      */
     RenderedLightObjectInstance.prototype.getDefaultWidth = function () {
-      return this._pixiObject.width;
+      return this._radius * 2;
     };
 
     /**
      * Return the height of the instance, when it's not resized.
      */
     RenderedLightObjectInstance.prototype.getDefaultHeight = function () {
-      return this._pixiObject.height;
+      return this._radius * 2;
```

Comment:
We don't allow to have a non-circular light?

Reply:
I think it should be `this._pixiObject` width and height dimensions here.
The new `this._radius * 2` doesn't fit with the comment two line above.

Reply:
Well, lights can't be resized - and even if they could, this is supposed to return the default size?